validation/citrus-validation-json/src/main/java/org/citrusframework/functions/core/EscapeJsonFunction.java

```diff
@@ -0,0 +1,31 @@
+package org.citrusframework.functions.core;
+
+import org.citrusframework.context.TestContext;
+import org.citrusframework.functions.Function;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Objects;
+
+import static org.apache.commons.lang3.StringEscapeUtils.escapeJson;
+/**
+ * This function takes a JSON string as input and escapes all double quotes within it.
+ *
+ * <p>The input must be a single non-empty string containing a valid JSON, wrapped in double quotes.</p>
+ * <p>If the input is invalid (null, empty, or contains more than one string), an {@link IllegalArgumentException} will be thrown.</p>
+ *
+ * <p>Example input: <code>"{\"mySuperJson\": \"valium\"}"</code></p>
+ * <p>Example output: <code>"{\\\"mySuperJson\\\": \\\"valium\\\"}"</code></p>
+ *
+ */
+public class EscapeJsonFunction implements Function {
+    @Override
+    public String execute(List<String> list, TestContext testContext) {
+        if (Objects.isNull(list) || list.size() != 1 || list.get(0).isEmpty()) {
```

Comment:
Still a list with  a null elementcould be passed in here. So better check this:
```suggestion
        if (Objects.isNull(list) || list.size() != 1 || StringUtils.isEmpty(list.get(0))) {
```

Reply:
added the null-ckeck which is null safe;)